ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/inventory/group_vars/TENANTS_NETWORKS.yml

```diff
@@ -28,6 +28,15 @@ tenants:
             name: Tenant_A_OP_Zone_2
             tags: [opzone]
             ip_address_virtual: 10.1.11.1/24
+            bgp:
+              structured_config:
+                # Will be merged with 'redistribute routes learned'
+                redistribute_routes:
+                  - connected
```

Comment:
For PR testing let's use the following:
      redistribute_routes:
        - router-mac system
      no_redistribute_routes:
        - host-route